### PR TITLE
Bosses of Mass Destruction 翻译提交

### PR DESCRIPTION
**Pull request by nageih**
resolve #2742

### CODE REVIEW

File: projects/1.16-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,171 @@
+{
+  "entity.bosses_of_mass_destruction.comet": "彗星",
```
Comment (luoxueyasha) [inline projects/1.16-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json:2]: 1.16的中文与英文key不对应，可以解释一下原因吗

**Review by luoxueyasha — CHANGES_REQUESTED** [review]

File: projects/1.16-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json (hunk 1 above)
Comment (nageih) [reply projects/1.16-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json:2]: 1.16直接用的1.18的语言文件，我对比了下，1.18的语言文件已经包含了1.16的所有key，而且内容一样，所以就偷懒了

Comment (luoxueyasha) [reply projects/1.16-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json:2]: 行吧，记得改的时候三个版本都改

File: projects/1.18-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+  "item.bosses_of_mass_destruction.mob_ward.tooltip" : "放置后，周围64格将不再生成生物",
+
+  "block.bosses_of_mass_destruction.monolith_block" : "爆破增幅器",
+  "item.bosses_of_mass_destruction.monolith_block.tooltip_0" : "放置后，周围64格的爆炸威力将增加30%",
+  "item.bosses_of_mass_destruction.monolith_block.tooltip_1" : "效果不可叠加",
+
+  "block.bosses_of_mass_destruction.levitation_block" : "飞腾台座",
+  "item.bosses_of_mass_destruction.levitation_block.tooltip" : "放置后，在7x7范围内获得创造飞行效果",
+
+  "item.bosses_of_mass_destruction.earthdive_spear.tooltip" : "可以短距离传送穿过固体方块",
```
Comment (luoxueyasha) [inline projects/1.18-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json:50]: 读起来有点怪
```suggestion
  "item.bosses_of_mass_destruction.earthdive_spear.tooltip" : "可以传送穿过一定厚度的固体方块",
```

File: projects/1.18-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+  "subtitles.bosses_of_mass_destruction.gauntlet_cast" : "下界铁掌：施法",
+  "subtitles.bosses_of_mass_destruction.gauntlet_spin_punch" : "下界铁掌：旋转",
+  "subtitles.bosses_of_mass_destruction.gauntlet_laser_charge" : "下界铁掌：充能",
+  "subtitles.bosses_of_mass_destruction.gauntlet_death" : "下界铁掌：死亡",
+  "subtitles.bosses_of_mass_destruction.gauntlet_clink" : "下界铁掌：叮当",
+  "subtitles.bosses_of_mass_destruction.soul_star" : "灵魂之星：下落",
+
+  "subtitles.bosses_of_mass_destruction.energy_shield" : "黑曜石柱：开启护盾",
+  "subtitles.bosses_of_mass_destruction.obsidilith_prepare_attack": "黑曜石柱：充能",
+  "subtitles.bosses_of_mass_destruction.spike" : "冰霜：",
```
Comment (luoxueyasha) [inline projects/1.18-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json:76]: ？

File: projects/1.18-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+  "text.autoconfig.bosses_of_mass_destruction.option.lichConfig.health" : "生命值",
+  "text.autoconfig.bosses_of_mass_destruction.option.lichConfig.missile" : "投掷物",
+  "text.autoconfig.bosses_of_mass_destruction.option.lichConfig.missile.statusEffectId" : "状态效果",
+  "text.autoconfig.bosses_of_mass_destruction.option.lichConfig.missile.statusEffectDuration" : "状态效果持续时间",
+  "text.autoconfig.bosses_of_mass_destruction.option.lichConfig.missile.statusEffectPotency" : "状态效果强度",
+  "text.autoconfig.bosses_of_mass_destruction.option.lichConfig.missile.damage" : "伤害",
+  "text.autoconfig.bosses_of_mass_destruction.option.lichConfig.comet" : "彗星",
+  "text.autoconfig.bosses_of_mass_destruction.option.lichConfig.comet.explosionStrength" : "爆炸强度",
+  "text.autoconfig.bosses_of_mass_destruction.option.lichConfig.comet.destroysBlocks" : "破坏方块",
+  "text.autoconfig.bosses_of_mass_destruction.option.lichConfig.summonMechanic" : "灵魂之星凋落",
```
Comment (luoxueyasha) [inline projects/1.18-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json:127]: 是`掉落`还是`凋落`？

File: projects/1.18-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+  "text.autoconfig.bosses_of_mass_destruction.option.lichConfig.summonMechanic.isEnabled" : "启用",
+  "text.autoconfig.bosses_of_mass_destruction.option.lichConfig.summonMechanic.entitiesThatCountToSummonCounter" : "可掉落的生物",
```
Comment (luoxueyasha) [inline projects/1.18-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json:129]: 可掉落……？

**Review by luoxueyasha — CHANGES_REQUESTED** [review]

File: projects/1.18-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json (hunk 3 above)
Comment (nageih) [reply projects/1.18-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json]: 已修复

File: projects/1.18-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json (hunk 4 above)
Comment (nageih) [reply projects/1.18-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json]: 是`掉落`，已修复

File: projects/1.18-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json (hunk 5 above)
Comment (nageih) [reply projects/1.18-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json:129]: 已改为`可掉落灵魂之星的生物`

**Review by luoxueyasha — DISMISSED** [review]
image
 一佰昏！特别是那几个成就名翻得拍手叫绝

**Comment by nageih** [comment]
需不需要resolve老旧pr

**Comment by luoxueyasha** [comment]
> 需不需要resolve老旧pr

走正常7+7流程就行

**Comment by nageih** [comment]
ok

File: projects/1.16-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+  "entity.bosses_of_mass_destruction.gauntlet" : "下界铁掌",
+  "entity.bosses_of_mass_destruction.void_blossom" : "虚空之花",
+  "entity.bosses_of_mass_destruction.spore_ball" : "孢子",
+  "entity.bosses_of_mass_destruction.petal_blade" : "花瓣利刃",
+
+  "block.bosses_of_mass_destruction.obsidilith_end_frame" : "黑曜石祭坛",
+  "block.bosses_of_mass_destruction.obsidilith_rune" : "黑曜石符文",
+  "block.bosses_of_mass_destruction.sealed_blackstone" : "尘封黑石砖",
+  "block.bosses_of_mass_destruction.gauntlet_blackstone" : "远古雕刻黑石",
+  "block.bosses_of_mass_destruction.chiseled_stone_altar" : "錾制石祭坛",
```
Comment (Cactusstudent) [inline projects/1.16-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json:16]:
```suggestion
  "block.bosses_of_mass_destruction.chiseled_stone_altar" : "錾制石头祭坛",
```
原版的石xx已经在很早之前改为了石头xx

Comment (Cactusstudent) [inline projects/1.16-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json:16]:
```suggestion
  "block.bosses_of_mass_destruction.chiseled_stone_altar" : "錾制石头祭坛",
```
原版的石xx已经在很早之前改为了石头xx

File: projects/1.16-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+  "item.bosses_of_mass_destruction.crystal_fruit.tooltip" : "获得短暂的生命恢复与抗性提升效果",
+  "item.bosses_of_mass_destruction.charged_ender_pearl" : "充能末影珍珠",
+  "item.bosses_of_mass_destruction.charged_ender_pearl.tooltip" : "可重复使用",
+  "item.bosses_of_mass_destruction.charged_ender_pearl.tooltip2" : "传送后给予短暂的抗性提升效果",
+  "item.bosses_of_mass_destruction.crafting_material.tooltip" : "制作材料",
+  "item.bosses_of_mass_destruction.brimstone_nectar" : "硫磺花蜜",
+  "item.bosses_of_mass_destruction.brimstone_nectar.tooltip" : "使用后重置附近的Boss结构",
+
+  "block.bosses_of_mass_destruction.mob_ward" : "镇压之杖",
+  "item.bosses_of_mass_destruction.mob_ward.tooltip" : "放置后，周围64格将不再生成生物",
```
Comment (Cactusstudent) [inline projects/1.16-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json:41]: 原文：**When placed, mobs won't spawn in any 64 block direction**

这里游戏内考证一下？

File: projects/1.16-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json [hunk 8]
```diff
@@ -0,0 +1,171 @@
...
+  "subtitles.bosses_of_mass_destruction.missile_prepare" : "暗夜巫妖：准备发射霜星",
+  "subtitles.bosses_of_mass_destruction.missile_shoot" : "暗夜巫妖：发射霜星",
+  "subtitles.bosses_of_mass_destruction.comet_prepare" : "暗夜巫妖：准备发射彗星",
+  "subtitles.bosses_of_mass_destruction.comet_shoot" : "暗夜巫妖：发射彗星",
+  "subtitles.bosses_of_mass_destruction.blue_fireball_land" : "霜星：着地",
+
+  "subtitles.bosses_of_mass_destruction.void_blossom_burrow" : "虚空之花：钻洞",
+  "subtitles.bosses_of_mass_destruction.void_blossom_spike" : "突刺：伸出",
+  "subtitles.bosses_of_mass_destruction.spore_impact" : "毒素：爆裂",
+  "subtitles.bosses_of_mass_destruction.petal_blade" : "花瓣利刃",
```
Comment (Cactusstudent) [inline projects/1.16-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json:101]: 这里需要一个原文

Comment (Cactusstudent) [inline projects/1.16-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json:101]: 这里需要一个原文

Comment (nageih) [reply projects/1.16-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json:101]: 原文`Petals`

File: projects/1.16-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json (hunk 7 above)
Comment (nageih) [reply projects/1.16-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json:41]: 试完了，功能确实与描述不符，实测是只会禁止怪物生成，但原文又是mob而不是monster，是遵照原文还是改了

Comment (Cactusstudent) [reply projects/1.16-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json:41]: 可以改

**Comment by mamaruo** [comment]
我对比了“#3597”和“#2742”的翻译，结果如下：

### 两者皆有且中文有差异

<details><summary>点击展开/收起</summary>
<p>

| **键** | **英文** | **#3597** | **#2742** |
|---|---|---|---|
| entity.bosses_of_mass_destruction.comet | Comet | 彗星 | 坠星 |
| entity.bosses_of_mass_destruction.lich | Night Lich | 暗夜巫妖 | 夜歌 |
| entity.bosses_of_mass_destruction.blue_fireball | Blue Fireball | 蓝色火球 | 幽焰弹 |
| entity.bosses_of_mass_destruction.soul_star | Soul Star | 灵魂之星 | 魂星 |
| entity.bosses_of_mass_destruction.obsidilith | Obsidilith | 黑曜石柱 | 龙晶结构体 |
| entity.bosses_of_mass_destruction.gauntlet | Nether Gauntlet | 下界铁掌 | 下界巨拳 |
| entity.bosses_of_mass_destruction.petal_blade | Petal Blade | 花瓣利刃 | 落英 |
| block.bosses_of_mass_destruction.sealed_blackstone | Sealed Blackstone Bricks | 尘封黑石砖 | 密封的黑石 |
| block.bosses_of_mass_destruction.gauntlet_blackstone | Ancient Carved Blackstone | 远古雕刻黑石 | 錾制远古黑石 |
| block.bosses_of_mass_destruction.void_blossom_block | Void Blossom Boss Spawner | 虚空之花Boss刷怪笼 | 虚空之花刷怪笼 |
| block.bosses_of_mass_destruction.void_lily | Void Lily | 虚空百合 | 空无百合 |
| block.bosses_of_mass_destruction.void_lily.tooltip | Where does the pollen lead? | 花粉将带你到何处？ | 它的花粉将指引你至何方？ |
| block.bosses_of_mass_destruction.vine_wall | Bramble Wall | 灌木墙 | 灌木屏障 |
| item.bosses_of_mass_destruction.soul_star.tooltip | Where does it lead? | 它将带你到何处？ | 它将带你至何方？ |
| item.bosses_of_mass_destruction.ancient_anima | Ancient Anima | 上古灵魂 | 古代灵魂 |
| item.bosses_of_mass_destruction.crystal_fruit.tooltip | Grants brief regen and resistance | 获得短暂的生命恢复与抗性提升效果 | 获得短暂的再生和抗性提升效果 |
| item.bosses_of_mass_destruction.charged_ender_pearl | Charged Ender Pearl | 充能末影珍珠 | 充能完毕的末影珍珠 |
| item.bosses_of_mass_destruction.charged_ender_pearl.tooltip2 | resistance after teleport | 传送后给予短暂的抗性提升效果 | 传送后获得短暂的抗性提升效果 |
| item.bosses_of_mass_destruction.crafting_material.tooltip | Crafting material | 制作材料 | 用于制作的材料 |
| item.bosses_of_mass_destruction.brimstone_nectar.tooltip | Resets nearby boss structures on use | 使用后重置附近的Boss结构 | 使用后重置附近的boss生成结构 |
| block.bosses_of_mass_destruction.mob_ward | Staff of Suppression | 镇压之杖 | 威压之杖 |
| item.bosses_of_mass_destruction.mob_ward.tooltip | When placed, mobs won't spawn in any 64 block direction | 放置后，周围64格将不再生成怪物 | 放置后，64格范围内将不再生成怪物 |
| block.bosses_of_mass_destruction.monolith_block | Blast Amplifier | 爆破增幅器 | 激化装置 |
| item.bosses_of_mass_destruction.monolith_block.tooltip_0 | When placed, increases explosion power by 30% in any | 放置后，周围64格的爆炸威力将增加30% | 使周围一切爆炸威力提高30% |
| item.bosses_of_mass_destruction.monolith_block.tooltip_1 | 64 block direction\. This effect cannot stack | 效果不可叠加 | 激化范围：64格，无法叠加 |
| block.bosses_of_mass_destruction.levitation_block | Table of Elevation | 飞腾台座 | 反重力装置 |
| item.bosses_of_mass_destruction.levitation_block.tooltip | When placed, grants creative flight in a 7x7 column | 放置后，在7x7范围内获得创造飞行效果 | 使7x7范围内的玩家能够自由飞行 |
| item.bosses_of_mass_destruction.earthdive_spear.tooltip | Grants short range teleport through solid blocks | 可以传送穿过一定厚度的固体方块 | 能够带使用者短距离穿过实体方块 |
| item.bosses_of_mass_destruction.earthdive_spear | Earthdive Spear | 潜地矛 | 地心投矛 |
| advancements.nether.gauntlet_defeat.title | Through the Gauntlet | 击穿掌眼 | 打得一拳开 |
| advancements.nether.gauntlet_defeat.description | Defeat the Nether Gauntlet Boss | 击败下界铁掌Boss | 击败下界巨拳 |
| advancements.end.obsidilith_defeat.title | Obsidian Obliterator | 黑曜石毁灭者 | 返璞归真 |
| advancements.end.obsidilith_defeat.description | Defeat the Obsidilith Boss | 击败黑曜石柱Boss | 击败龙晶结构体（黑曜石古称龙晶） |
| advancements.adventure.night_lich_defeat.title | Night Vanquisher | 暗夜消退 | 夜之颂歌 |
| advancements.adventure.night_lich_defeat.description | Defeat the Night Lich Boss | 击败暗夜巫妖Boss | 击败夜歌 |
| advancements.adventure.void_blossom_defeat.title | Extreme Weeding | 寸草不生 | 极限除草 |
| advancements.adventure.void_blossom_defeat.description | Defeat the Void Blossom Boss | 击败虚空之花Boss | 击败虚空之花 |
| subtitles.bosses_of_mass_destruction.gauntlet_idle | Gauntlet Breathes | 下界铁掌：呼吸 | 下界巨拳：呼吸 |
| subtitles.bosses_of_mass_destruction.gauntlet_hurt | Gauntlet Hurts | 下界铁掌：受伤 | 下界巨拳：受伤 |
| subtitles.bosses_of_mass_destruction.gauntlet_cast | Gauntlet Casts | 下界铁掌：施法 | 下界巨拳：投掷 |
| subtitles.bosses_of_mass_destruction.gauntlet_spin_punch | Gauntlet Spins | 下界铁掌：旋转 | 下界巨拳：转体 |
| subtitles.bosses_of_mass_destruction.gauntlet_laser_charge | Gauntlet Charges Up | 下界铁掌：充能 | 下界巨拳：充能 |
| subtitles.bosses_of_mass_destruction.gauntlet_death | Gauntlet Dies | 下界铁掌：死亡 | 下界巨拳：死亡 |
| subtitles.bosses_of_mass_destruction.gauntlet_clink | Gauntlet Clinks | 下界铁掌：叮当 | 下界巨拳：叮当 |
| subtitles.bosses_of_mass_destruction.soul_star | Soul Star Falls | 灵魂之星：下落 | 魂星：下落 |
| subtitles.bosses_of_mass_destruction.energy_shield | Obsidilith Shields | 黑曜石柱：开启护盾 | 龙晶结构体：开启护盾 |
| subtitles.bosses_of_mass_destruction.obsidilith_prepare_attack | Obsidilith Charges Up | 黑曜石柱：充能 | 龙晶结构体：充能 |
| subtitles.bosses_of_mass_destruction.spike | Frost Erupts | 冰霜：聚爆 | 冰霜能量：震爆 |
| subtitles.bosses_of_mass_destruction.spike_indicator | Frost Gathers | 冰霜：汇聚 | 冰霜能量：汇聚 |
| subtitles.bosses_of_mass_destruction.wave_indicator | Fire Gathers | 火焰：汇聚 | 火焰能量：汇聚 |
| subtitles.bosses_of_mass_destruction.obsidilith_burst | Energy Erupts | 能量：聚爆 | 能量：泄露 |
| subtitles.bosses_of_mass_destruction.obsidilith_teleport | Obsidilith Teleports | 黑曜石柱：传送 | 龙晶结构体：传送 |
| subtitles.bosses_of_mass_destruction.obsidilith_death | Obsidilith Dies | 黑曜石柱：死亡 | 龙晶结构体：死亡 |
| subtitles.bosses_of_mass_destruction.obsidilith_hurt | Obsidilith Hurts | 黑曜石柱：受伤 | 龙晶结构体：受损 |
| subtitles.bosses_of_mass_destruction.obsidilith_wave | Fire Erupts | 火焰：聚爆 | 火焰能量：爆裂 |
| subtitles.bosses_of_mass_destruction.lich_death | Lich Dies | 暗夜巫妖：死亡 | 夜歌：死亡 |
| subtitles.bosses_of_mass_destruction.lich_hurt | Lich Hurts | 暗夜巫妖：受伤 | 夜歌：受伤 |
| subtitles.bosses_of_mass_destruction.lich_teleport | Lich Teleport | 暗夜巫妖：传送 | 夜歌：传送 |
| subtitles.bosses_of_mass_destruction.rage_prepare | Lich Rages | 暗夜巫妖：愤怒 | 夜歌：发怒 |
| subtitles.bosses_of_mass_destruction.minion_summon | Phantom Appears | 幻翼：出现 | 幻翼：被召唤 |
| subtitles.bosses_of_mass_destruction.minion_rune | Rune Appears | 符文：出现 | 符文：被召唤 |
| subtitles.bosses_of_mass_destruction.teleport_prepare | Lich Prepares Teleport | 暗夜巫妖：准备传送 | 夜歌：准备传送 |
| subtitles.bosses_of_mass_destruction.missile_prepare | Lich Prepares Iceballs | 暗夜巫妖：准备发射霜星 | 夜歌：准备发射霜星 |
| subtitles.bosses_of_mass_destruction.missile_shoot | Lich Shoots Iceballs | 暗夜巫妖：发射霜星 | 夜歌：发射霜星 |
| subtitles.bosses_of_mass_destruction.comet_prepare | Lich Prepares Comet | 暗夜巫妖：准备发射彗星 | 夜歌：准备发射坠星 |
| subtitles.bosses_of_mass_destruction.comet_shoot | Lich Launches Comet | 暗夜巫妖：发射彗星 | 夜歌：发射坠星 |
| subtitles.bosses_of_mass_destruction.void_blossom_burrow | Blossom Burrows | 虚空之花：钻洞 | 虚空之花：挖掘 |
| subtitles.bosses_of_mass_destruction.void_blossom_spike | Spikes | 突刺：伸出 | 虚空之花：刺击 |
| subtitles.bosses_of_mass_destruction.spore_impact | Poison Explodes | 毒素：爆裂 | 毒药：聚爆 |
| subtitles.bosses_of_mass_destruction.petal_blade | Petals | 花瓣利刃 | 落英：下落 |
| subtitles.bosses_of_mass_destruction.spore_prepare | Poison Bubbling | 毒素：酝酿 | 毒药攻击：准备 |
| subtitles.bosses_of_mass_destruction.spike_wave_indicator | Ground Trembles | 大地：震动 | 大地：震颤 |
| subtitles.bosses_of_mass_destruction.spore_ball_land | Poison Lands | 药水：着地 | 毒药：落地 |
| subtitles.bosses_of_mass_destruction.void_spike_indicator | Spikes Gather | 突刺：集聚 | 尖刺：聚拢 |
| subtitles.bosses_of_mass_destruction.void_blossom_fall | Blossom Falls | 虚空之花：凋落 | 虚空之花：陨落 |
| subtitles.bosses_of_mass_destruction.earthdive_spear_throw | Spear Throws | 潜地矛：掷出 | 投矛：投掷 |
| subtitles.bosses_of_mass_destruction.charged_ender_pearl | Pearl Lands | 充能末影珍珠：着地 | 珍珠：落地 |
| subtitles.bosses_of_mass_destruction.brimstone | Brimstone releases | 硫磺：喷出 | 硫磺：释放而出 |
| text.autoconfig.bosses_of_mass_destruction.title | All changes require mc restart | 所有的更改都需要重启MC | 需要重启游戏 |
| text.autoconfig.bosses_of_mass_destruction.category.Lich | Lich | 暗夜巫妖 | 夜歌 |
| text.autoconfig.bosses_of_mass_destruction.option.lichConfig.eternalNighttime | Apply Eternal Nightime | 启用永夜 | 激活永夜 |
| text.autoconfig.bosses_of_mass_destruction.option.lichConfig.idleHealingPerTick | Idle Healing per Tick | 脱战每刻自愈量 | 脱战状态每刻的自愈量 |
| text.autoconfig.bosses_of_mass_destruction.option.lichConfig.missile | Missile | 投掷物 | 霜弹 |
| text.autoconfig.bosses_of_mass_destruction.option.lichConfig.missile.damage | Damage | 伤害 | 攻击力 |
| text.autoconfig.bosses_of_mass_destruction.option.lichConfig.comet | Comet | 彗星 | 坠星 |
| text.autoconfig.bosses_of_mass_destruction.option.lichConfig.summonMechanic.entitiesThatCountToSummonCounter | Entities that can drop | 可掉落灵魂之星的生物 | 可掉落的生物 |
| text.autoconfig.bosses_of_mass_destruction.option.lichConfig.summonMechanic.numEntitiesKilledToDropSoulStar | Kills per soul star | 每颗所需的击杀数 | 每颗掉落所需的击杀数 |
| text.autoconfig.bosses_of_mass_destruction.category.Obsidilith | Obsidilith | 黑曜石柱 | 龙晶结构体 |
| text.autoconfig.bosses_of_mass_destruction.option.obsidilithConfig.idleHealingPerTick | Idle Healing per Tick | 脱战每刻自愈量 | 脱战状态每刻的自愈量 |
| text.autoconfig.bosses_of_mass_destruction.option.obsidilithConfig.arenaGeneration | Arena Generation | 角斗场生成 | 斗技场生成 |
| text.autoconfig.bosses_of_mass_destruction.category.Gauntlet | Gauntlet | 下界铁掌 | 下界巨拳 |
| text.autoconfig.bosses_of_mass_destruction.option.gauntletConfig.idleHealingPerTick | Idle Healing per Tick | 脱战每刻自愈量 | 脱战状态每刻的自愈量 |
| text.autoconfig.bosses_of_mass_destruction.option.gauntletConfig.spawnAncientDebrisOnDeath | Spawn Ancient Debris On Death | 死亡后生成远古残骸 | 死亡后生成下界残骸 |
| text.autoconfig.bosses_of_mass_destruction.option.gauntletConfig.arenaGeneration | Arena Generation | 角斗场生成 | 斗技场生成 |
| text.autoconfig.bosses_of_mass_destruction.option.voidBlossomConfig.idleHealingPerTick | Idle Healing per Tick | 脱战每刻自愈量 | 脱战状态每刻的自愈量 |
| text.autoconfig.bosses_of_mass_destruction.option.voidBlossomConfig.arenaGeneration | Arena Generation | 角斗场生成 | 斗技场生成 |

</p>
</details>

### 只在 #3597 中有

🚫未找到

### 只在 #2742 中有

🚫未找到

**Review by luoxueyasha — CHANGES_REQUESTED** [review]

**Comment by nageih** [comment]
@luoxueyasha 已与#2742作者协商完成，决定使用我这个pr，并resovle他的，有一些新的更新看commit就好，那个commit消息的pr号打错了，问题应该不大（

**Comment by IceDragonFla** [comment]
#2742 作者报道，使用本pr，部分译名已经协商完成。

**Pull request merged**